stash/test/api/faucet.api.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, it, beforeEach, expect, vi } from 'vitest'
+import supertest from 'supertest';
+import app from '../../src/app';
+import * as faucetService from '../../src/service/FaucetService.js'
+import { ForbiddenException } from '../../src/error/Exception.js'
+
+
+describe('FaucetController', () => {
+  const toAddress = '0x00000000000000000000000000000000'
+  const captcha = '10000000-aaaa-bbbb-cccc-000000000001'
```

Comment:
This captcha token is test one publicly available on hcaptcha website [here](https://docs.hcaptcha.com/#test-key-set-publisher-or-pro-account) but we can move it to sops if we need. @goncer opinion? stay or move to sops?